src/Sound.ts

```diff
@@ -510,34 +449,36 @@ export class Sound
-     * @return {PIXI.sound.IMediaInstance|Promise<PIXI.sound.IMediaInstance>} The sound instance,
+     * @method play
+     * @instance
+     * @param {String} alias - The unique name of the sound sprite.
+     * @param {object} data - Either completed function or play options.
+     * @param {number} data.start - Time when to play the sound in seconds.
+     * @param {number} data.end - Time to end playing in seconds.
+     * @param {number} data.speed - Override default speed, default to the Sound's speed setting.
+     * @param {Function} callback - Callback when completed.
+     * @return {IMediaInstance|Promise<PIXI.sound.IMediaInstance>} The sound instance,
      *        this cannot be reused after it is done playing. Returns a Promise if the sound
      *        has not yet loaded.
      */
     public play(alias: string, callback?: CompleteCallback): IMediaInstance | Promise<IMediaInstance>;
 
     /**
      * Plays the sound.
-     * @method PIXI.sound.Sound#play
-     * @param {PIXI.sound.Sound~completeCallback|object} options Either completed function or play options.
-     * @param {number} [options.start=0] Time when to play the sound in seconds.
-     * @param {number} [options.end] Time to end playing in seconds.
-     * @param {String} [options.sprite] Play a named sprite. Will override end, start and speed options.
-     * @param {number} [options.speed] Override default speed, default to the Sound's speed setting.
-     * @param {number} [options.volume] Current volume amount for instance.
-     * @param {boolean} [options.muted] Override default muted, default to the Sound's muted setting.
-     * @param {boolean} [options.loop] Override default loop, default to the Sound's loop setting.
-     * @param {PIXI.sound.Sound~completeCallback} [options.complete] Callback when complete.
-     * @param {PIXI.sound.Sound~loadedCallback} [options.loaded] If the sound isn't already preloaded, callback when
+     * @method play
+     * @instance
+     * @param {Sound~completeCallback|object} options - Either completed function or play options.
+     * @param {number} [options.start=0] - Time when to play the sound in seconds.
+     * @param {number} [options.end] - Time to end playing in seconds.
+     * @param {String} [options.sprite] - Play a named sprite. Will override end, start and speed options.
+     * @param {number} [options.speed] - Override default speed, default to the Sound's speed setting.
+     * @param {number} [options.volume] - Current volume amount for instance.
+     * @param {boolean} [options.muted] - Override default muted, default to the Sound's muted setting.
+     * @param {boolean} [options.loop] - Override default loop, default to the Sound's loop setting.
+     * @param {Sound~completeCallback} [options.complete] - Callback when complete.
+     * @param {Sound~loadedCallback} [options.loaded] - If the sound isn't already preloaded, callback when
```

Comment:
As above, can these be moved to the interface itself?

Reply:
Yes